### DISCUSSION TITLE
Tasmota32 don't compile

### DISCUSSION THREAD

**Discussion by HelioBoechat** [arendst/Tasmota#15928] [Q&A] [answered]
Hi,

I need to integrate my equipaments to IOT and chosse Tasmota to do so and I aim write a especific driver.
So I decided to begin by simply build the standart Tasmota-Master.
I installed PlatformIO 6.0.2 (I see the Jason2866 advise), with platforms Espressif 32 and Espressif 8266 installed and download last Tasmota-Master branch.
In 'PlatformIO Project Task' correctly appear all environments.

Building by 'PlatformIO Project Task' (no .ini files changed, shouldn't work ?):
All Tasmota... environments (boards ESP8266) build correctly.
All Tasmota32... environments (boards ESP32) finish in the same error, after compiling the various libs of framework-arduinoespressif32:

Linking .pio\build\tasmota32\firmware.elf
c:/users/cliente/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld.exe: cannot open map file D:/Tasmota-master/.pio/build/tasmota32/firmware.map: No such file or directory
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\tasmota32\firmware.elf] Error 1

firmware.map file was not generated.

I delete Tasmota folder, uninstall PlatformIO, unistall VSC and users related files, install everything again without errors and had the same behave.
Environments with Esp8266 boards compile, environments with Esp32 boards has the above error.

Using Tasmota develpoment brench has the same behave.
Comment and uncomment the environments in platformio_override.ini has the same behave.
Editing directly platformio.ini (ensuring build_flags = -Wl,-Map,firmware.map) has the same behave

I'm digging through community posts for days but didn't find anything that could help me.

I know I must be making a ridiculous mistake, but I don't find it.
Somebody could help this fresh newbe in Tasmota?

**Comment by barbudor** [comment]
Hi
Generally I don't install platformio standalone but install Vscode with the platformIO extension and let the extension install everything on the background

If you already have the platformIO extension you may try to
- close vscode
- delete '.pio' and '.cache' in the tasmota folder
- delete the whole '.platformio' folder in your home directory
- start vscode and let the pio extension download platformio
- close and restart vscode
- build tasmota

**Comment by HelioBoechat** [comment, marked as answer]
Thanks barbudor for your answer,

I had already done this, but I carefully redid all the steps.
Do not work.

But you confirmed it must be an installation issue.
After many attempts, I move Tasmota folder outside Dropbox folder and everything works !!!

I've never had these issues with Dropbox on other platforms or frameworks.
Leaving and learning..

Looking for the reason why ESP8266 boards works and ESP32 do not work is counterproductive, at least for me.

Thank you very much.

**Reply by barbudor** [reply]
Cool you found the reason although I don't see why Dropbox would interfere....
A few years ago I had my work directory under dropbox too to avoid syncing multiple PC through github. I can't recal such problems.
Now I've given up and go through Github should I move from one PC to another.

**Reply by Jason2866** [reply]
The max path length can be a issue with windows when using cloud storage
